setup.py

```diff
@@ -5,6 +5,7 @@
 with open("README.md") as f:
     long_description = f.read()
 
+__version__ = "0.0.0"
```

Comment:
What's the rationale behind this change?

Reply:
Yeah, if it's going to stay, I'd drop a `magic thing to get python-semantic-release working, please do not touch` comment or something like it, just to protect our future selves.